messages/agent.preview.md

```diff
@@ -12,8 +12,7 @@ When the session concludes, the command asks if you want to save the API respons
 
 Find the agent's API name in its main details page in your org's Agent page in Setup.
 
-Before you use this command, you must complete these steps:
------------------------------------------------------------
+## Before you use this command, you must complete these steps:
```

Comment:
```suggestion
Before you use this command, you must complete these steps:
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

Reply:
@WillieRuemmele , I think VScode or the GH UI automatically changes text like `=======` in MD files to `##`, but I don't want to make this a header because it messes up the `--help`.  So let's try a bunch of tildes instead.

Reply:
ok, there's some automatic linting rule that keeps changing this, I'll try your suggestion